python/health-check/README.md

```diff
@@ -0,0 +1,54 @@
+### mgr-health-check
+
+A tool providing dashboard, metrics and logs from an Uyuni server supportconfig to visualise its health status.
```

Comment:
If I'm not mistaken, we use en-US, so this should be with  Z (visualise -> visualize)
```suggestion
A tool providing dashboard, metrics and logs from an Uyuni Server supportconfig to visualize its health status.
```